apps/condo/domains/meter/schema/MeterReportingPeriod.js

```diff
@@ -66,46 +66,37 @@ const MeterReportingPeriod = new GQLListSchema('MeterReportingPeriod', {
             kmigratorOptions: { null: true, on_delete: 'models.CASCADE' },
         },
 
-        start: {
+        startAt: {
             schemaDoc: 'Starting day of the month for sending a push about the need to submit meter readings',
             type: Integer,
             defaultValue: 20,
             isRequired: true,
             hooks: {
                 validateInput: async ({ context, operation, resolvedData }) => {
                     if (operation === 'create' || operation === 'update') {
-                        if (resolvedData.start > 31 || resolvedData.start < 1) {
+                        if (resolvedData.startAt > 31 || resolvedData.startAt < 1) {
                             throw new GQLError(ERRORS.INVALID_START, context)
                         }
                     }
                 },
             },
         },
 
-        finish: {
+        finishAt: {
             schemaDoc: 'Finish day of the month for sending a push about the need to submit meter readings',
             type: Integer,
             defaultValue: 25,
             isRequired: true,
             hooks: {
                 validateInput: async ({ context, operation, resolvedData }) => {
                     if (operation === 'create' || operation === 'update') {
-                        if (resolvedData.finish > 31 || resolvedData.finish < 1) {
+                        if (resolvedData.finishAt > 31 || resolvedData.finishAt < 1) {
```

Comment:
What if value will be negative?

Reply:
-1 < 1 there will be an error. In addition, at the frontend, this is implemented by a select with numbers, and not by an input. Therefore, a negative number is only possible with a bug or creating a period in the mutation / admin panel

Reply:
Could you please add check for negative values. Another weak point is an admin panel.
It's better to include data logic into schema itself, rather than UI.

Reply:
Approvement diverges from our implicit standard. Up to you to decide what to do.